Patch to allow us to specify alternate autogen, columns and getdefs binaries
when running tests.

This patch has been sent upstream.

--- autogen-5.16.2/agen5/bootstrap.dir.orig	2015-08-11 09:49:20.142168670 -0700
+++ autogen-5.16.2/agen5/bootstrap.dir	2015-08-11 09:51:33.172606217 -0700
@@ -40,9 +40,9 @@
 {
     builddir=$(pwd)
     srcdir=${srcdir:-$builddir}
-    AGexe=$(command -v autogen)
-    CLexe=$(command -v columns)
-    GDexe=$(command -v getdefs)
+    AGexe=${AGexe:-$(command -v autogen)}
+    CLexe=${CLexe:-$(command -v columns)}
+    GDexe=${GDexe:-$(command -v getdefs)}
 
     test -x "${AGexe}" -a -x "${GDexe}" -a -x "${CLexe}" || \
         die "autogen is required"
--- autogen-5.16.2/config/bootstrap.orig	2015-08-11 09:52:09.296125796 -0700
+++ autogen-5.16.2/config/bootstrap	2015-08-11 09:53:13.481511016 -0700
@@ -72,7 +72,7 @@
 
     #  Check for AutoGen version 5.
     #
-    AGexe=$(command -v autogen)
+    AGexe=${AGexe:-$(command -v autogen)}
     case "${AGexe}" in
     ( /* )
         v=$($AGexe --version || :)
@@ -87,8 +87,8 @@
         ;;
     esac
 
-    GDexe=$(command -v getdefs)
-    CLexe=$(command -v columns)
+    GDexe=${GDexe:-$(command -v getdefs)}
+    CLexe=${CLexe:-$(command -v columns)}
     test -x "${GDexe}" -a -x "${CLexe}" || \
         die "autogen support programs are missing"
     export AGexe GDexe CLexe
